boards/hifive1/README.md

```diff
@@ -21,11 +21,37 @@ Running in QEMU
 The HiFive1 application can be run in the QEMU emulation platform, allowing quick and easy testing.
 
 QEMU can be started with Tock using the following arguments (in Tock's top-level directory):
+
+```bash
+$ qemu-system-riscv32 -M sifive_e -kernel $TOCK_ROOT/target/riscv32imac-unknown-none-elf/release/hifive1.elf  -nographic
 ```
-qemu-system-riscv32 -M sifive_e -kernel target/riscv32imac-unknown-none-elf/release/hifive1.elf  -nographic
+
+Or with the `qemu` make target:
+
+```bash
+$ make qemu
```

Comment:
In both cases, should the command invocation be duplicated, or should the documentation just direct folks to call make and let what make executes be self-documenting (worried the doc will fall out of sync / the command here provides relatively little value, unless it's changed often?)